build/cargo.gni

```diff
@@ -0,0 +1,23 @@
+# Runs a script to compile a local Cargo package and all of its dependencies
+template("cargo_build") {
+  action(target_name) {
+    rust_deps_version = "0.1.0"
+
+    script = "//brave/script/cargo.py"
+
+    # make sure rebuilds when files change
+    forward_variables_from(invoker, "*")
+
+    outputs = [output]
+
+    args = [
+      "--rustup_bin=" + rebase_path("//brave/build/rustup/" + rust_deps_version + "/bin", root_build_dir),
```

Comment:
I guess we don't really need this since we always know that it's rustup_home + bin. Removing it will simplify the rust_deps_version thing I mentioned below because we can just pass the base path here

Reply:
👍